wee_slack.py

```diff
@@ -113,7 +113,7 @@
         "history": None,
         "join": None,
         "leave": None,
-        "mark": None,
+        "mark": "subscriptions.thread.mark",
```

Comment:
Does this work for you? For me it just returns:

```
{
  "ok": false,
  "error": "not_allowed_token_type"
}
```

Reply:
Where do you see this?  I thought I had it working, but then I backed out some changes before I pushed, and it doesn't seem to work now.

Reply:
I see it if I call it manually, or if I add this to your branch and navigate away from a thread:

```
def handle_subscriptionsthreadmark(json, eventrouter, team, channel, metadata):
    print(json)
```

Additionally, the thread is not marked as read. Did you confirm that the thread was marked as read when you thought you had it working?

Reply:
I thought I did, but my testing procedure (checking in the slack web client) may have not been entirely accurate.

Reply:
OK, I get the same result.  This may not be possible at the moment, then.

Reply:
It turns out that if you use a token pulled out of the web client ([as described here](https://github.com/yuya373/emacs-slack#how-to-get-token)), calling `subscriptions.thread.mark` works.

Reply:
I don't think we should require users to use those tokens, but maybe we should provide the functionality for users that do?

Reply:
Would we want to unconditionally try these APIs in case they work?  Or should we try to check to see if the user is using a more privileged API token?  This would also allow us to use the thread subscribe API.

Reply:
Hm, I'm not sure. Just add it unconditionally for now, we can add a condition later if we want.

Reply:
OK, I added that and a couple of other things that rely on the special tokens.